Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG ARCH=amd64
+ARG DISTRO=debian
+ARG DISTRO_VERSION=buster
+ARG VERSION=0.1
+
+FROM netdata/builder:${DISTRO}_${DISTRO_VERSION} AS build
+
+ARG ARCH
+ARG DISTRO
+ARG DISTRO_VERSION
+ARG VERSION
+
+ENV ARCH=$ARCH
+ENV DISTRO=$DISTRO
+ENV DISTRO_VERSION=$DISTRO_VERSION
+ENV VERSION=$VERSION
+
+WORKDIR /netdata
+COPY . .
+
+RUN /build.sh
+
+FROM ${DISTRO}:${DISTRO_VERSION} AS runtime
+
+ARG ARCH
+ARG DISTRO
+ARG DISTRO_VERSION
+ARG VERSION
+
+ENV ARCH=$ARCH
+ENV DISTRO=$DISTRO
+ENV DISTRO_VERSION=$DISTRO_VERSION
+ENV VERSION=$VERSION
+
+COPY .dockerfiles/install.sh /install.sh
+COPY .dockerfiles/test.sh /test.sh
+
+COPY --from=build /netdata/artifacts /artifacts
+
+RUN /install.sh || exit 1
```

Comment:
Is `|| exit 1` needed? Is it not true that if a RUN command fails then the build fails?

Reply:
Yes it does. Fixed :)